Slim/Schema/Genre.pm

```diff
@@ -9,6 +9,12 @@ use Slim::Schema::ResultSet::Genre;
 
 use Slim::Utils::Misc;
 use Slim::Utils::Log;
+use Slim::Utils::Prefs;
+
+my $myClassicalGenreMap;
+my $myClassicalGenreIds;
+my $tagSeparator;
+my $tagSeparatorLoaded;
```

Comment:
These seem to be unused?

Reply:
Ah yes, when I was considering loading the tag separator into Genre rather than amend splitTag. You can get rid.